### PR TITLE
Rearrange command line options

### PR DESCRIPTION
**Pull request by PeterJCLaw**
Tidy up the existing command line options to have slightly cleaner implementations and move to using `BaseCommand` and `argparse` rather than `NoArgsCommand` and `optparse`.

### CODE REVIEW

File: django_lightweight_queue/management/commands/queue_runner.py [hunk 1]
```diff
@@ -85,6 +86,10 @@ def touch_filename(name):
         get_middleware()
         log.info("Loaded middleware")
 
+        # Configuration overrides
+        if num_only_queue_workers is not None:
+            app_settings.WORKERS[only_queue] = num_only_queue_workers
```
Comment (tomokas) [inline django_lightweight_queue/management/commands/queue_runner.py]: Should we check `only_queue` is not None here too?

File: django_lightweight_queue/management/commands/queue_runner.py [hunk 2]
```diff
@@ -1,39 +1,40 @@
...
+        parser.add_argument('--touchfile', action='store', dest='touchfile', default=None,
+            help="touch(1) the specified file after running a job.")
+        parser.add_argument('--machine', action='store', dest='machine_number', default='1',
+            help="Machine number, for parallelism")
+        parser.add_argument('--of', action='store', dest='machine_count', default='1',
+            help="Total number of machines running the queues")
+        parser.add_argument('--only-queue', action='store', default=None,
+            help="Only run the given queue, useful for local debugging")
+        parser.add_argument('--num-queue-workers', action='store', default=None,
+            help="The number of queue workers to run, only valid when running a specific queue")
```
Comment (tomokas) [inline django_lightweight_queue/management/commands/queue_runner.py]: For the handling of `num_only_queue_workers` where we check it's an int and positive -- can we do that here instead via a validation function?

Comment (PeterJCLaw) [inline django_lightweight_queue/management/commands/queue_runner.py]: I suppose we could, though it seems pointless given that the entire option is removed by #15.

File: django_lightweight_queue/management/commands/queue_runner.py (hunk 1 above)
Comment (PeterJCLaw) [inline django_lightweight_queue/management/commands/queue_runner.py]: We don't need to -- the validation of the command line flags means that if `num_only_queue_workers` is not `None` then we have a valid `only_queue`. I appreciate that that's a little indirect, but given that we remove `num_only_queue_workers` entirely in #15 I'm not worried.

**Pull request merged**